src/app/components/Home/Home.jsx

```diff
@@ -15,9 +15,11 @@ const Home = ({ sortBy }) => (
 
     <div className="content-wrapper">
       <div className="nypl-column-three-quarters nypl-column-offset-one">
```

Comment:
all the divs... but now here's one you can remove! that `<div className="search home">` why dat? search can be a sibling of `nypl-lead`.